src/script/cpp_api/s_security.cpp

```diff
@@ -590,6 +590,15 @@ bool ScriptApiSecurity::checkPath(lua_State *L, const char *path,
 		}
 	}
 
+	// Allow read-only access to game.conf
+	if (!write_required) {
+		if (const Server *server = dynamic_cast<const Server *>(gamedef)) {
```

Comment:
given how central games are to MT I think it's better to move `getGameSpec` to `IGameDef` than to break the abstraction here